script/api.py

```diff
@@ -0,0 +1,105 @@
+import os
+
+import psycopg2
+from flask import Flask, jsonify, request, session
+
+app = Flask(__name__)
+
+hostname = os.getenv('POSTGRES_HOST')
+database = os.getenv('POSTGRES_DB')
+username = os.getenv('POSTGRES_USER')
+
+port_id = 5432
+conn = psycopg2.connect(
+    host=hostname,
+    dbname=database,
+    user=username,
+    port=port_id
+)
+
+cur = conn.cursor()
+
+@app.route('/signup', methods=['POST'])
+def signup():
+    """SIGNUP API
+    Parameters:
+    username (String): Username of user
+    email (String): Email of user
+    password (String): Password of user
+
+    Returns:
+        Http Response: {
+            String : Message 
+            Integer : Status Code
+        }
+    """
+
+    username=request.json.get('username')
+    email = request.json.get('email')
+    password = request.json.get('password')
+    cur.execute("SELECT email FROM users WHERE email='{}'".format(email))
+    user_with_email=cur.fetchone()
+
+    if user_with_email :
+        return jsonify({"Message": "User already exist"}),400
+        
+    cur.execute("INSERT INTO users (name,email,password) VALUES ('{}','{}',crypt('{}',gen_salt('bf')))".format(username,email,password))
+    conn.commit()
+    return jsonify({"Message": "User Registered Successfully"})
```

Comment:
```suggestion
    return jsonify({"message": "User Registered Successfully"})
```
We usually keep the keys in a json response in lower case. Please update all the instances